### PR TITLE
Fixes #13562: file_keys_values_present does not suport some separators

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/13562

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: tree/20_cfe_basics/ncf_lib.cf [hunk 1]
```diff
@@ -491,9 +491,10 @@ bundle edit_line ncf_maintain_keys_values(v, sep) {
       # Keys and values may contain regexps. Escape them for replace_pattern matching.
       "ve[${index}]" string => escape("${${v}[${index}]}");
       "ke[${index}]" string => escape("${index}");
+      "separator"    string => escape("${sep}");
```
Comment (amousset) [inline tree/20_cfe_basics/ncf_lib.cf]: line 505 should be fixed too

Comment (amousset) [reply tree/20_cfe_basics/ncf_lib.cf]: 550 too

Comment (ncharles) [reply tree/20_cfe_basics/ncf_lib.cf]: insert_lines should not have the escape

**Pull request merged**